lib/ui/home/widget/home_survey_page.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/gen/assets.gen.dart';
+import 'package:flutter_survey/theme/app_dimensions.dart';
+import 'package:flutter_survey/ui/widget/dimmed_background.dart';
+import 'package:flutter_survey/ui/widget/next_button.dart';
+
+const _maxLines = 2;
```

Comment:
@Tuubz Why don't we declare these variables inside the class `HomeSurveyPage`?

Reply:
In Kotlin, we define `const` variables at the top of the class. 

Do you think they should be inside?

Reply:
For me, if this constant is only used by one class it should be inside that class, you can go with Kotlin conventional way as you like.

Reply:
Oh, just to clarify. 

This is a private const, in Datter it's done by adding `_` in front of the variable name 🚀

Reply:
@Tuubz IMO, this is too specific to the Page item's title text. We can use the value `2` directly in the view or give a better name for this const. `_maxLines = 2` is too vague and does not make sense to me without a context.